<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>489</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>627</width>
    <height>489</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>627</width>
    <height>489</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Trabalho 1</string>
  </property>
  <property name="windowIcon">
   <iconset theme="Mario">
    <normaloff>favicon.ico</normaloff>favicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonVai">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Vai!!!</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTamanho">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Std</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1000000</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxTipo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Std</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Crescente</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Decrescente</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bagunçado</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxMetodo">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Bubble</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Inserction</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Selection</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Merge</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Quick</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Heap</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-20</y>
      <width>671</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>41656272113.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>JoaoPauloBorges Development™ -  Todos os direitos reservados © - 2017</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>pushButtonVai</zorder>
   <zorder>comboBoxTamanho</zorder>
   <zorder>comboBoxTipo</zorder>
   <zorder>comboBoxMetodo</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
